Builds/containers/packaging/dpkg/debian/copyright

```diff
@@ -0,0 +1,86 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: rippled
+Source: https://github.com/ripple/rippled 
+
+Files: *
+Copyright: 2012-2019 Ripple Labs Inc.
+
+License:   __UNKNOWN__
```

Comment:
I believe this was detected by `debmake`, probably because we have a bit of eclectic collection of licenses in our source. I'm someone knows definitively how to declare this kind of mix of licenses, I'm happy to change it.